rpc_jobs/merge_pull_request.yml

```diff
@@ -0,0 +1,29 @@
+- job:
+    name: 'Merge-Pull-Request'
+    project-type: pipeline
+    concurrent: false
+    properties:
+      - build-discarder:
+          days-to-keep: 30
+    parameters:
+      - rpc_gating_params
+      - validating-string:
+          name: pr_repo
+          regex: "[a-zA-Z0-9-]+/[a-zA-Z0-9._-]+"
+          msg: "The release repository must be supplied."
+      - validating-string:
+          name: pr_number
+          regex: "[0-9]+"
+          msg: "The number of the pull request to merge is required."
+      - validating-string:
+          name: retries
+          regex: "[0-9]+"
+          default: "1"
```

Comment:
Any reason we default this to 1 here but 0 in the python lib itself?

Reply:
It seemed reasonable for the function to just make the request by default but allow retries if desired.